package.json

```diff
@@ -29,7 +29,10 @@
     "vue-omnibox": "^0.3.7",
     "vue-sliding-pagination": "^1.3.2",
     "vuedraggable": "^2.24.3",
-    "vuex": "^3.6.2"
+    "vuex": "^3.6.2",
+    "webpack": "^5.75.0",
```

Comment:
This needs to be reverted again

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/91/commits/a4221b334ca9d377eb1728c3aad8a9463c86349e